server/auth/types/authentication_type.ts

```diff
@@ -96,77 +96,76 @@ export abstract class AuthenticationType implements IAuthenticationType {
   }
 
   public authHandler: AuthenticationHandler = async (request, response, toolkit) => {
-    // skip auth for APIs that do not require auth
+    // Skip authentication for APIs that do not require it
     if (this.authNotRequired(request)) {
       return toolkit.authenticated();
     }
 
     const authState: OpenSearchDashboardsAuthState = {};
-
-    // if browser request, auth logic is:
```

Comment:
I think this comment block is helpful. wdyt about keeping this here?

Reply:
I removed it in preference for the code comments in the rest of that function which say basically the same thing; but I can put it back for a summary if you wish.